### PR TITLE
iperf: add page

### PR DESCRIPTION
**Pull request by codetheweb**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/iperf.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+> Measure network bandwidth between computers.
+
+- Run on server:
+
+`iperf -s`
+
+- Run on client:
+
+`iperf -c <server's address>`
```
Comment (agnivade) [inline pages/common/iperf.md]: Please use tokens for server address - `{{server_host}}`

**Comment by codetheweb** [comment]
Sorry for all these errors; this is my first time contributing to tldr.

**Comment by agnivade** [comment]
FYI - You can also run the linter locally with `npm test`.

**Comment by codetheweb** [comment]
Looks good, thanks.

File: pages/common/iperf.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+> Measure network bandwidth between computers.
+
+- Run on server:
+
+`iperf -s`
+
+- Run on client:
+
+`iperf -c {{server_address}}`
```
Comment (sbrl) [inline pages/common/iperf.md]: What is meant by `server_address`? Is it just an IP address, or a port number too?

Comment (codetheweb) [reply pages/common/iperf.md]: Just an IP address.  If no port number is provided, iperf uses the default port 5001.

Comment (sbrl) [reply pages/common/iperf.md]: Ah, ok. I think this should be fine then :-)

**Pull request merged**